devU-api/scripts/populate-db.ts

```diff
@@ -0,0 +1,170 @@
+        const assign302_2 = await SendPOST("/assignments", JSON.stringify({
+            courseId: course302, name: "Sprint 2", startDate: "2024-02-17", dueDate: "2024-03-09", endDate: "2024-03-09T00:00:01-0500", gradingType: "manual", categoryName: "Sprints",
+            description: "The Second Sprint", maxFileSize: 10000000, disableHandins: false
+        }))
+        SendPOST("/assignments", JSON.stringify({
+            courseId: course302, name: "Sprint 3", startDate: "2024-03-09", dueDate: "2024-04-06", endDate: "2024-04-06T00:00:01-0500", gradingType: "manual", categoryName: "Sprints",
+            description: "The Third Sprint", maxFileSize: 10000000, disableHandins: false
+        }))
+
+
+        //AssignmentProblems - ROUTE NOT IMPLEMENTED
+        // const assign312_quiz_q1 = await SendPOST("/assignment-problems", JSON.stringify({
+        //     assignmentId: assign312_quiz, problemName: "q1", maxScore: 5
+        // }))
+        // const assign312_quiz_q2 = await SendPOST("/assignment-problems", JSON.stringify({
+        //     assignmentId: assign312_quiz, problemName: "q2", maxScore: 5
+        // }))
+
+
+        //Submissions
+        const submission_billy_302_1 = await SendPOST("/submissions", JSON.stringify({
+            courseId: course302, assignmentId: assign302_1, userId: userBilly, content: "I finished all my tasks for sprint 1!", type: "text", 
+            submitterIp: "127.0.0.1", submittedBy: userBilly
+        }))
+        SendPOST("/submissions", JSON.stringify({
+            courseId: course302, assignmentId: assign302_2, userId: userBilly, content: "I finished all my tasks for sprint 2!", 
+            type: "text", submitterIp: "127.0.0.1", submittedBy: userBilly
+        }))
+        const submission_bob_312_1 = await SendPOST("/submissions", JSON.stringify({
+            courseId: course312, assignmentId: assign312_1, userId: userBob, content: "jesse pleaseee can i have a good grade pleaseeee", type: "text",
+            submitterIp: "127.0.0.1", submittedBy: userBob
+        }))
+        SendPOST("/submissions", JSON.stringify({
+            courseId: course312, assignmentId: assign312_2, userId: userBob, content: "im begging you jesse give this one a good grade", type: "text",
+            submitterIp: "127.0.0.1", submittedBy: userBob
+        }))
+        // const submission_bob_312_quiz1 = await SendPOST("/submissions", JSON.stringify({
+        //     courseId: course312, assignmentId: assign312_quiz, userId: userBob, content: "B", type: "text", submitterIp: "127.0.0.1", submittedBy: userBob
+        // }))
+        
+        
+        //SubmissionProblemScores - ROUTE NOT IMPLEMENTED
+        // SendPOST("/submission-problem-scores", JSON.stringify({
+        //     submissionId: submission_bob_312_quiz1, assignmentProblemId: assign312_quiz_q1, score: 5, feedback: "Good job!"
+        // }))
+
+
+        //SubmissionScores - Swagger documentation is wrong, /submission-scores not /submission-score
```

Comment:
.. please update the docs instead of leaving this comment. Those docs are created from the submission score router and we missed the s's. Head to that file and update